core/tests/protractor_desktop/extensions.js

```diff
@@ -134,7 +134,7 @@ describe('Interactions', function() {
         explorationEditorMainTab.setInteraction.apply(
           null, [interactionId].concat(test.interactionArguments));
 
-        explorationEditorMainTab.addResponse.apply(null, [
+        explorationEditorMainTab.addResponse.apply(explorationEditorMainTab, [
```

Comment:
Can you explain the reason for this change?

Reply:
If the `explorationEditorMainTab` is not passed in, we will get `undefined` error https://travis-ci.org/oppia/oppia/jobs/596916385#L4852. 
I slightly changed the code in `addResponse` and if no context is passed in, this line will complain. 
https://github.com/oppia/oppia/blob/f99921188031c10c6609ef72d37e3a67de297a15/core/tests/protractor_utils/ExplorationEditorMainTab.js#L256